docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -306,3 +288,120 @@ Le code ci-dessus génère ce qui suit sur la page :
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### Structure du dossier
```

Comment:
```suggestion
### Arborescence de fichiers
```

I think it would make more sense to use ["Arborescence de fichiers"](https://fr.wikipedia.org/wiki/Arborescence#/media/Fichier:AVCHD_actual_file_structure.svg) here like you did below to match the English version.